src/renderers/dom/__tests__/ReactDOMProduction-test.js

```diff
@@ -178,6 +178,9 @@ describe('ReactDOMProduction', () => {
     expect(function() {
       class Component extends React.Component {
         render() {
+          // FIXME: undefined is a valid return type in Fiber, at least in the
+          // current implementation. It's treated as empty. I don't know why we
+          // sometimes get false positives.
```

Comment:
We should fix this in Fiber. This should not be ok. I have noticed that this sometimes gets a false positive when I run the `record-tests` script with the ReactDOMFeatureFlag turned on locally. cc @spicyj